src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocmissingleadingasterisk/Example1.java

```diff
@@ -14,34 +15,34 @@
  *   int value = 0;
  * </pre>
  */
-class JavaStyle {} // ok
+class JavaStyle {}
 
 /** Valid Scala-style comment.
   * Some description here.
   **/
-class ScalaStyle {} // ok
+class ScalaStyle {}
 
 /** **
   * Asterisks on first and last lines are optional.
   * */
-class Asterisks {} // ok
+class Asterisks {}
 
 /** No asterisks are required for single-line comments. */
-class SingleLine {} // ok
+class SingleLine {}
 
-/** // violation on next blank line, javadoc has lines without leading asterisk.
+/** // violation below
 
  */
 class BlankLine {}
 
 /** Wrapped
-    single-line comment */ // violation, javadoc has lines without leading asterisk.
+    single-line comment */ // violation
```

Comment:
Please do not remove the reason for the violation from messages. We provide examples of violations and their reasons to help the user understand the module and how it functions.

Please do it for all the violations, ok messages.

Reply:
I have added the messages for violations.
Doing the same for ok messages is making the file look clumsy, so I reverted those. Suggest if otherwise

Reply:
Check has multiple violations ,so we need violation message partly be in comment line.

We can skip violation message if Check has only single violation message.